server/models.py

```diff
@@ -82,3 +83,27 @@ class ResourcesNeeded(db.Model):
     id: Mapped[int] = mapped_column(primary_key=True, autoincrement=True)
     action_id: Mapped[int] = mapped_column(ForeignKey("action.id"), nullable=False)
     resource_id: Mapped[int] = mapped_column(ForeignKey("resource.id"), nullable=False)
+
+
+class WebUser(db.Model):
+    username: Mapped[str] = mapped_column(primary_key=True)
+    password: Mapped[str]
+
+    def is_authenticated(self):
+        return True
+
+    def is_active(self):
+        return True
+
+    def is_anonymous(self):
+        return False
```

Comment:
Where are these used? Why are the responses hard-coded? Just curious.

Reply:
They are used by Flask-Login: https://flask-login.readthedocs.io/en/latest/#your-user-class. Hardcoded because afaik we don't use them (yet) but they are needed.